tests/cases/standalone/common/order/nulls_first.sql

```diff
@@ -8,9 +8,11 @@ SELECT * FROM test ORDER BY i NULLS FIRST, j NULLS FIRST;
 
 SELECT * FROM test ORDER BY i NULLS LAST, j NULLS FIRST;
 
-SELECT i, j, row_number() OVER (PARTITION BY i ORDER BY j NULLS FIRST) FROM test ORDER BY i NULLS FIRST, j NULLS FIRST;
+-- TODO(ruihang): The following two SQL will fail under distributed mode with error
```

Comment:
Create an issue to track it, it's a serious issue.

Reply:
https://github.com/GreptimeTeam/greptimedb/issues/1702